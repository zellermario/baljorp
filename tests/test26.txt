create SJ sj1 0
create SJ sj2 0
addNeighbor sj1 1 sj2
addNeighbor sj2 2 sj1

create eszkimo e sj2
create jegesmedve m sj1

move m 1
pass m