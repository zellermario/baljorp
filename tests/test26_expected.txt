Fields:
SJ sj1 (0) - object: x
	neighbours: [(1,sj2)]
SJ sj2 (0) - object: x
	neighbours: [(2,sj1)]

Buildings:

Actors:
> Eszkimo eszkimo (heat: 0) on sj2
  Inventory: []
> Jegesmedve m (heat: 1) on sj2
  Inventory: []